src/components/App.css

```diff
@@ -1,7 +1,26 @@
 .calculator {
+  z-index: 1;
   position: absolute;
   top: 40px;
   left: 70px;
   right: 0;
   bottom: 0;
 }
+
+.calculator-burst {
+  left: 271px;
+  animation: fade-in 0.5s;
```

Comment:
Per offline discussion, remote `fade-in` animations for all classes and combine `.calculator-burst` `.calculator-settings` together,

Reply:
grouped like classes together, removed animations, changed z-indexes per above comment